### PR TITLE
Playwright Tests

### PR DESCRIPTION
**Pull request by satchelbaldwin**
* Add playwright testing for core beaker functionality

### Done:
* Basic agent conversation: "say hello" -> ensure agent response says hello
* Basic agent tool usage: "use python code to..." and ensure the result is printed
* Code running via keyboard shortcuts and toolbar button
* File panel UI
* Rehydrating conversation state and context from a saved notebook to run edge cases long into workflows
* CI actions for running tests 

### Todo:
* Test cases over long notebooks
* Workflows:
  * Selecting a workflow
    * Agent/User workflow selection and using the tool call sidebar to ensure it's correct
    * Loading workflows in a context 
* Sidebars:
  * Coverage of sidebar UI elements working as intended such as inspecting a PDF 
* Chat history: Ensuring summarization and context cleaning
* Inline plotting tests in code cells
* UI updates for code cell state

### CODE REVIEW

File: beaker-vue/src/components/notebook/BeakerNotebookToolbar.vue [hunk 1]
```diff
@@ -243,6 +243,14 @@ const handleExport = (format: string, mimetype: string) => {
         });
     }
 
+    if (format === "full_context_notebook") {
+        const history = await session.executeAction("get_agent_history", {}).done;
+        console.log(history);
+        tempNotebook.content.metadata.message_history = {
+            full_history: history.content.return
+        }
```
Comment (mattprintz) [inline beaker-vue/src/components/notebook/BeakerNotebookToolbar.vue]: This introduces two brand new terms, "message_history" and "full_history", which I believe to be essentially synonyms with the existing term "chat_history".

Is there any reason to not just use `chat_history` here and bypass the nesting. Something like `tempNotebook.content.metadata.chat_history = history.content.return`?
